plugins/idea/src/main/java/org/robovm/idea/RoboVmPlugin.java

```diff
@@ -223,12 +231,16 @@ public void run() {
         });
     }
 
+    private static String getNowString() {
+        return LocalDateTime.now().format(formatter);
+    }
+
     public static void logInfo(Project project, String format, Object... args) {
-        log(project, ConsoleViewContentType.SYSTEM_OUTPUT, "[INFO] " + format, args);
+        log(project, ConsoleViewContentType.SYSTEM_OUTPUT, "[INFO] " + getNowString() + format, args);
```

Comment:
do we really need these timestamps at every day basics ? As it useful during development but just mess an output when you don't care. Probably it might be useful to use it in case when RoboVM is running in development mode

Reply:
Yes, we need them. It enables everybody to see, when there's a problem. Additionally it has the potential to motivate other people to contribute. E.g. somebody could be motivated to contribute logic to skip the slow interface compilation or asset creation processes (example from my machine):

```
[INFO] 10:22:33.071 /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool ...
[INFO] 10:22:35.726 /Applications/Xcode.app/Contents/Developer/usr/bin/actool ...
[INFO] 10:22:38.892 /* com.apple.actool.compilation-results */
```

So, let's keep them.